src/cmd/run.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import type FirefoxProfile from 'firefox-profile';
```

Comment:
@kumar303 I would prefer if we can keep the flow `import/export type`s clearly separated from the regular ES6 imports/exports,
mostly because I think that it would make much more clear that we are not importing/exporting code but flow types.

How about moving all the `import/export type` grouped after the group of ES6 module imports?

e.g.
``` 
import * as defaultFirefox from '../firefox';
import defaultFirefoxConnector from '../firefox/remote';
...
import defaultSourceWatcher from '../watcher';

// Import and export Flow types
import type FirefoxProfile from 'firefox-profile';
```